### PR TITLE
Api endpoint for new city request form. closes #117

### LINKED ISSUE #117: Zpracovani formulare zapojeni nove obce
### Popis
Je potreba vytvorit endpoint pro zpracovani udaju z formulare, ktery identifikuje zajem o zapojeni nove obce do CityVizoru. Konkretne je potreba sbirat udaje (viz UI navrh):

1) email
2) jmeno
3) true/false souhlas se zasilanim informaci.

Soucasne je potreba, aby vysledky byly dostupne v realnem case, v rozumne podobe, bez technickych znalosti. (preferovana varianta je ulozeni do Google Docs).

### Technologie

Endpoint by mel byt psany v Kotlinu jako soucast server-kotlin projektu:

https://github.com/cityvizor/cityvizor/blob/master/server-kotlin/src/main/Application.kt

### Screenshot odpovidajiciho UI (FE neni soucast issue).

obrázek

### PR DESCRIPTION
**Pull request by michalsvec**
This still need a bit more work mainly to configure proper deployment and google document settings. 

Application now takes parameter with a google api credentials json file. Is that okay for final deployment? 

What's the best way to configure various google sheet identifiers? For now it's hardcoded into the application, but should be probably in some sort of configuration.

### CODE REVIEW

File: server-kotlin/src/city_request/CityRequest.kt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class CityRequest(
+    @JsonProperty("city")
+    val city: String,
+
+    @JsonProperty("email")
+    val email: String,
+
+    @JsonProperty(value = "name", required = false)
+    val name: String,
```
Comment (martinwenisch) [inline server-kotlin/src/city_request/CityRequest.kt]: Pokud neni name required daval bych prednost pouziti nullable String?

File: server-kotlin/src/city_request/CityRequest.kt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    @JsonProperty("subscribe")
+    val subscribe: Boolean,
+
+    @JsonProperty("gdpr")
+    val gdpr: Boolean
+) {
+    override fun toString(): String {
```
Comment (martinwenisch) [inline server-kotlin/src/city_request/CityRequest.kt]: Je nejaky duvod pouzivat jiny toString() nez defaultni z data class? Prijde mi to jako zbytecny boilerplate.

File: server-kotlin/src/city_request/CityRequestRouter.kt [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import io.ktor.response.respond
+import io.ktor.routing.Route
+import io.ktor.routing.Routing
+import io.ktor.routing.post
+import io.ktor.routing.route
+
+fun Routing.cityRequestRouter(): Route {
+    return route("/api/v2/service/cityrequest") {
+        post("/") {
+            Patterns.ask(
```
Comment (martinwenisch) [inline server-kotlin/src/city_request/CityRequestRouter.kt]: Pokud je to fire&forget, tak je lepsi pouzit tell

CityRequestActors.get()?.tell(call.receive(CityRequest::class), ActorRef.noSender())

File: server-kotlin/src/city_request/CityRequestActors.kt [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        if (!this::actorSystem.isInitialized) {
+            return null
+        }
+
+        return actorSystem.actorSelection("/user/" + CityRequestActor::class.simpleName)
+    }
+
+    private fun createGoogleSheetsStore(): GoogleSheets {
+        val documentId = "12SM9gbOlKG2OnplHryuhx9oGAQh3JHGumSZVU7g4Gzo"
+        val listName = "Seznam"
```
Comment (martinwenisch) [inline server-kotlin/src/city_request/CityRequestActors.kt]: Asi by stalo za to pouzit application.conf (resolvovany config da se klidne pridat jako static val do Application class).

**Comment by michalsvec** [comment]
Second iteration ready

**Pull request merged**